src/js/mixins/AppActionsHandlerMixin.jsx

```diff
@@ -54,6 +54,14 @@ var callErrorDialog = function (settings) {
   });
 };
 
+var isStatefulApp = function (app) {
+  return app.container != null &&
+    app.container.volumes != null &&
+    app.container.volumes
+      .filter(volume => volume.persistent != null || volume.external != null)
```

Comment:
You could use `volumes.some(volume => volume.persistent != null || volume.external != null)` to check if there is at least one persistent volume.